mkdocs-project-dir/docs/howto.md

```diff
@@ -29,12 +35,15 @@ You will need to create an entry for the host you are connecting to with the set
 In newer versions of PuTTY, it looks like this.
 
 *TODO: new putty*
+ <!--- Win 10 PuTTY looks same  --->
 
 You will then be asked to enter your username and password. Only enter your username, not `@legion.rc.ucl.ac.uk`. The password field will remain entirely blank when you type in to it - it does not show placeholders to indicate you have typed something. 
+<!---          @<system_name>.rc.ucl.ac.uk  --->
 
```

Comment:
Is there a reason that we do not comply with 

MD009 - Trailing spaces
MD012 - Multiple consecutive blank lines
MD013 - Line length
MD022 - Headers should be surrounded by blank lines
MD031 - Fenced code blocks should be surrounded by blank lines
MD032 - Lists should be surrounded by blank lines

From e.g., https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md

What breaks compatibility with what, if we respect the rules above?

Reply:
Mostly laziness I suspect.  Certainly in my case.

Reply:
With respect to MD013: rewrapping pages with a change can make it more difficult to see what was actually changed.